### DISCUSSION TITLE
Does marked support inserting videos

### DISCUSSION THREAD

**Discussion by thusimon** [markedjs/marked#2698] [Q&A] [answered]
Hi,
I tried something like `![video](http://foo.com/test.mp4)`, but no luck.
Is it possible to support inserting videos?
Thanks a lot!

**Comment by UziTech** [comment, marked as answer]
see https://github.com/markedjs/marked/issues/362#issuecomment-1126700852

**Reply by thusimon** [reply]
That helps a lot, thanks!

**Comment by unphased** [comment]
I just tested and it does work to just plop a video HTML tag right in the markdown document for what it's worth. Bringing video content in via iframe is certainly a fine way to embed videos hosted elsewhere, but not aligning with my minimalist preferences...

**Reply by UziTech** [reply]
Yes all HTML is valid markdown. Any HTML tags will be output as is.
